.vsts-ci.yml

```diff
@@ -345,17 +351,48 @@ stages:
 
   - template: /eng/common/templates/jobs/source-build.yml
 
-  - template: /src/SourceBuild/Arcade/eng/common/templates/job/source-build-create-tarball.yml
+- stage: Build_Linux
```

Comment:
Originally, I wasn't making any assumptions about the configurations we run in PRs so I assumed we can have SDK content tests in PRs as well and then I needed this dependency.

I also found out that when linking the 2 pipelines (installer, vmr), you can say which stages matter:
```yaml
  - pipeline: installer-build-resource
    source: dotnet-installer-official-ci
    trigger:
      branches:
        include:
        - main
        - release/*
        - internal/release/*
      stages:
      - Build_Linux
      - VMR_Synchronization
```

So if we kept this as a separate stage, we could possibly start the VMR build ~30 minutes earlier when we already have the portable legs finished but not needing to wait for the other installer jobs. Do we care about that?

Reply:
The name is not great and I would put something else if we want to keep this.

Reply:
>Originally, I wasn't making any assumptions about the configurations we run in PRs so I assumed we can have SDK content tests in PRs as well and then I needed this dependency.

We don't want this dependency in PR.  We shouldn't be running the the SDK content tests in PR.  We want to avoid dependencies in order to keep the PR validation as short as possible given the VMR build is the longest leg.

> So if we kept this as a separate stage, we could possibly start the VMR build ~30 minutes earlier when we already have the portable legs finished but not needing to wait for the other installer jobs. Do we care about that?

I think we should avoid this separate stage as it messes with the clarity of the build definition.  It would not be readily apparent why this is needed.  Furthermore over time there is no guarantee this would actually allow the VMR build to be queued any earlier.